lib/v2/gov/router.js

```diff
@@ -92,4 +92,5 @@ module.exports = function (router) {
     router.get(/xinyi(\/[\w/-]+)?/, require('./xinyi/xinyi'));
     router.get('/xuzhou/hrss/:category?', require('./xuzhou/hrss'));
     router.get('/zhejiang/gwy/:category?/:column?', require('./zhejiang/gwy'));
+    router.get('/mof/bond/:category?', require('./mof/bond'));
```

Comment:
Please insert new entries in alphabetical order.

Reply:
fixed.